### PR TITLE
Done. Open to feedback

### PR DESCRIPTION
**Pull request by jasonbraganza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,6 @@ Requirements:
 - Python 3.6
 - And everything from the requirements.txt file.
 
-`python jdv.py` to run the program
\ No newline at end of file
+You’ll need a text file full of sites, one per line.
+  
+`python jdv.py 'filename.txt'` to run the program
```
Comment (kushaldas) [inline README.md:12]: The instruction says to have single quotes around the filename, is that correct?

File: jdv.py [hunk 2]
```diff
@@ -5,46 +5,63 @@
...
+
+
+def local_lookup(some_site):
+    site_ip = socket.gethostbyname(some_site).strip()
+    return site_ip
+
+
+def google_lookup(some_site):
+    google_lookup = requests.get(f"https://dns.google/resolve?name={some_site}")
+    site_ip = google_lookup.json()["Answer"][0]["data"].strip()
```
Comment (kushaldas) [inline jdv.py:56]: What if there are more than one IP as a result? Say mirrors.fedoraproject.org ?

File: jdv.py [hunk 3]
```diff
@@ -5,46 +5,63 @@
...
-                    print("All ip lookups match.")
-                elif local_ip == google_ip and not local_ip == cloudflare_ip:
-                    print("Local ip matches with Google, but not Cloudflare.")
-                elif local_ip == cloudflare_ip and not local_ip == google_ip:
-                    print("Local dns lookup matches with Cloudflare, but not Google.")
-                else:
-                    print(
-                        "Local dns lookup does not match with either Google or Cloudflare."
-                    )
+        if sys.argv[1]:
```
Comment (kushaldas) [inline jdv.py:20]: What if there is no `sys.argv[1]`? There will no exception, correct?

**Review by kushaldas — CHANGES_REQUESTED** [review]
A few questions.

File: README.md (hunk 1 above)
Comment (jasonbraganza) [reply README.md:12]: how do i differentiate that from the rest of the command? as in that is something variable? not literally filename.txt?

File: jdv.py (hunk 2 above)
Comment (jasonbraganza) [reply jdv.py:56]: aah, yes. i had left that out on purpose.
will work at getting the rest and comparing those too.
Thank you.

File: jdv.py (hunk 3 above)
Comment (jasonbraganza) [reply jdv.py:20]: if no, there will be an exception and the exception should catch it. i tried that.
or am i not doing this right?

**Pull request closed without merging**